sources/packages/web/src/components/fa-application/StringToHtmlParagraphs.vue

```diff
@@ -0,0 +1,28 @@
+<template>
```

Comment:
I think you can add a test for this component.

Reply:
about my review comment requesting that a test be added for the component you created – On 2nd thoughts, i think the logic inside the component is very simple so this may NOT need to be tested. So leave it to you.. If you want to add a test go ahead and will again REview the PR . If you dont, want to add the test you can consider the PR merged!

Reply:
I agree that a test could be added here, it would be nice to keep adding as much as possible.
I created the StringToHtmlParagraphs.spec.ts and fixed one test that was failing because a component was moved.